content/en/docs/deployment/releases.md

```diff
@@ -18,9 +18,9 @@ For more information, check out the [chapter on releases](https://adoptingerlang
 Add a `relx` section to your project's `rebar.config`:
 
 ```erlang
-{relx, [{release, {<release name>, "0.0.1"},
+{relx, [{release, {<release_name>, <release_vsn>},
```

Comment:
The extra generality here sadly loses the inline example of how a typical release version looks like. Hope that's okay!

Reply:
Yep that's fine since the table below defines it.